content/docs/_index.md

```diff
@@ -2,7 +2,8 @@
 title: Documentation
 linktitle: Docs
 meta_desc: Learn how to create, deploy, and manage infrastructure on any cloud using Pulumi's open source infrastructure as code SDK.
-noindex: true
+exclude_from_pulumi_search_index: true
+block_external_search_index: true
```

Comment:
Let’s remove `block_external_search_index` here. I think it’s ok for Google et. al. to index this page. But keep `exclude_from_pulumi_search_index` as this isn’t really that relevant of a page when searching within the site.

Reply:
Fixed